### DISCUSSION TITLE
Ability to have an overlay without blur

### DISCUSSION THREAD

**Discussion by lorenzsell** [add2cal/add-to-calendar-button#206] [Q&A] [answered]
Hello - not sure if this is the right place to post but I did some research and couldn't find anywhere else to ask.  I'm trying to understand if there's a way to have the overlay show up without a blur and to control the opacity of the overlay?

Also, is there a way to pass styling to the calendar options that show up?

Thank you.

**Comment by jekuer** [comment, marked as answer]
Regarding styling: 
You can simply load your own css instead of the provided one. Copy and adjust the one from the package.
However, a more convenient way is upcoming.

Regarding the background style:
You can specify `"background":false` to disable it.
